Library/Formula/cjdns.rb

```diff
@@ -19,6 +19,42 @@ def install
     (share+"test").install "build_darwin/test_testcjdroute_c" => "cjdroute_test"
   end
 
+  def caveats
+    s = <<-EOS.undent
+      !!!IMPORTANT!!!
+      Before using cjdns, you must generate a configuration file and find peers manually.
+      Once you have found eligible peers, use the following command to generate a config that is appropriate for Launchd
+        sudo bash -c "cjdroute --genconf | sed 's@\\"noBackground\\":0@\\"noBackground\\":1@' > /etc/cjdroute.conf"
```

Comment:
Can this not be run automatically? Also, can the configuration file not use `HOMEBREW_PREFIX/etc`?